### PR TITLE
gio: Fix nullability of various DBus method call related parameters

### PR DESCRIPTION
**Pull request by sdroege**
CC @sophie-h

### CODE REVIEW

File: gio/tests/dbus_peer.rs [hunk 1]
```diff
@@ -0,0 +1,169 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[test]
```
Comment (bilelmoussaoui) [inline gio/tests/dbus_peer.rs:3]: Wouldn't moving this to test::run be enough? so you don't need the duplicate function :)  otherwise lgtm

Comment (sdroege) [reply gio/tests/dbus_peer.rs:4]: Indeed, this can be untangled considerably. Thanks :)

File: gio/tests/dbus_peer.rs [hunk 2]
```diff
@@ -0,0 +1,159 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[cfg(unix)]
+#[test]
+fn test_gdbus_peer_connection() {
+    use gio::{
+        glib::{self, VariantTy},
+        prelude::SocketExt,
```
Comment (bilelmoussaoui) [inline gio/tests/dbus_peer.rs:8]: You have the prelude below, replace this with prelude::* and remove the other one

Comment (sdroege) [reply gio/tests/dbus_peer.rs]: Done

**Pull request merged**